### PR TITLE
Bug - retarget status save

### PR DESCRIPTION
**Pull request by tjmgd**
When the retarget status is changed and then saved, the change is not applied. This fixes the issue by setting the dirty flag to true on change.

### CODE REVIEW

File: Gems/EMotionFX/Code/EMotionFX/Source/AnimGraph.cpp [hunk 1]
```diff
@@ -902,6 +902,7 @@ namespace EMotionFX
 
     void AnimGraph::OnRetargetingEnabledChanged()
     {
+        SetDirtyFlag(true);
```
Comment (amzn-jillich) [inline Gems/EMotionFX/Code/EMotionFX/Source/AnimGraph.cpp:905]: Can we call AnimGraph::SetRetargetingEnabled(true); here instead and move the SetDirtyFlag(true) call to the SetRetargetingEnabled function?

File: Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp [hunk 2]
```diff
@@ -1072,10 +1072,10 @@ namespace AzToolsFramework
             RefreshAutocompleter();
         }
 
-        // When focus is lost, clear the field if necessary
+        // When focus is lost, revert to the selected asset
         if (!focus && m_incompleteFilename)
         {
-            HandleFieldClear();
+            SetSelectedAssetID(GetCurrentAssetID());
```
Comment (amzn-jillich) [inline Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp:1078]: Is this related to the retarget flag saving?

Comment (tjmgd) [reply Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp:1078]: This change has been implemented in the latest version of development so it should no longer be relevant. But the PR was submitted before that.

**Comment by lawsonamzn** [comment]
Is this valid, or should it be closed?

**Comment by amzn-jillich** [comment]
Closing the PR as this has been solved with another PR already. Thanks for the submission!

**Comment by tjmgd** [comment]
@amzn-jillich Great! You don't happen to have a link for that PR do you? No problem if not.
